src/App.tsx

```diff
@@ -1,10 +1,28 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [key, setKey] = useState('');
+  const handleKey = (event: KeyboardEvent) => {
+    setKey(event.key);
+  };
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  useEffect(() => {
+    document.addEventListener('keyup', handleKey);
+
+    return () => document.removeEventListener('keyup', handleKey);
+  }, []);
```

Comment:
```suggestion
  }, [key]);
```

Reply:
amazing! thank you for pointing it out !